front/src/router/index.ts

```diff
@@ -19,6 +20,12 @@ const router = createRouter({
       component: Booking,
       meta: { requiresAuth: true },
     },
+    {
+      path: '/reservas/reserva',
```

Comment:
A url deverá ter o id da reserva que o usuário clicou, sendo algo como: /reservas/reserva/2
para adicionar esse parâmetro numérico, você faz:
   ```suggestion
      path: '/reservas/reserva/:reservationId(\\d+)'',
```

Reply:
Mas n temos dados mockados então da onde sairia esse numero no momento?

Reply:
Esse param funcionará com qualquer número que você pôr, por enquanto